### PR TITLE
Added a bog page, descriptions to the feature section, and Rindra and Obeda in the Our team section.

### PR DESCRIPTION
**Pull request by dah000**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206091377865175

### CODE REVIEW

File: blog.html [hunk 1]
```diff
@@ -0,0 +1,336 @@
...
+					businesses in the home services industry. Whether you're a
+					homeowner seeking the perfect contractor or a business looking to
+					expand your customer base, Tenafli has you covered. Embrace the
+					future of home services, bid farewell to the hassle of finding
+					reliable professionals, and say hello to a seamless, efficient,
+					and enjoyable experience. Download the app today and
+					revolutionize the way you connect with home service providers.
+				</p>
+			</div>
+			<!-- <div class="blog-container fs-5">
```
Comment (andy-ratsirarson) [inline blog.html:195]: is this commented section needed? if not, let's delete it.

File: assets/img/team/Rindra.jpg
Comment (andy-ratsirarson) [inline assets/img/team/Rindra.jpg:1]: @rindra-rolando can you provide a profile with a one-color background to be put on our site.

Comment (andy-ratsirarson) [inline assets/img/team/Rindra.jpg:1]: @maharo-raj can you provide a profile as well. Thanks

**Review by andy-ratsirarson — CHANGES_REQUESTED** [review]

File: blog.html (hunk 1 above)
Comment (dah000) [reply blog.html]: No it's not. I removed it.

**Review by andy-ratsirarson — APPROVED** [review]

**Pull request merged**